### ISSUE TITLE
Set "approved" as default status for comment connection queries

### ISSUE THREAD

**Issue by tonimain** [wp-graphql/wp-graphql#171]
Comment query needs some rework. If you submit a comment as a guest author it"s also visible when calling comments {} query.

**Comment by jasonbahl** [comment]
@tonimain unapproved comments are returned in the WP_Comment_Query unless you tell it not to. 

Comment queries by default include "all" stati of comments: https://core.trac.wordpress.org/browser/tags/4.8/src/wp-includes/class-wp-comment-query.php#L293

If you were working on a UI where you wanted to only display "approved" comments, you could do that by passing a status argument to the query, much like the core "comments template" and core "comments" widgets do: 

https://github.com/WordPress/WordPress/blob/9c2f5c18ce887e0ec2a8366f1f5a273e8f33b5b8/wp-includes/comment-template.php#L1993

https://github.com/WordPress/WordPress/blob/1d34043e8cd2eead37048bc8aed379d57ff32408/wp-includes/widgets/class-wp-widget-recent-comments.php#L96-L99

So, in WPGraphQL, if you wanted to query just approved comments, you could do so like this: 

```
	$query = 'query getApprovedComments($where:commentArgs) {
	  comments(where:$where) {
	    edges{
	       node{ 
	         id
	         content
	       }
	    }
	  }
	}';

	$variables = [
		'where' => [
			'status' => 'approve',
		],
	];

	$result = do_graphql_request( $query, 'getApprovedComments', $variables );
```

====

So, with that said, I suppose we could make it so that the default value for `status` for GraphQL comment queries is set to "approved" and any other status would need a `$where` arg passed? I believe the REST API does that, so I could probably get behind that idea too.

**Comment by jasonbahl** [comment]
@tonimain I'm going to change the title of this issue to `set "approved" as default status for comment connection queries`

**Comment by tonimain** [comment]
sounds good in my ears. think it's a good way to handle it this way

**Comment by jasonbahl** [comment]
This was addressed by #174
